api-report/tree2.api.md

```diff
@@ -1031,6 +1031,9 @@ export class InvalidationToken {
 interface Invariant<in out T> extends Contravariant<T>, Covariant<T> {
 }
 
+// @alpha
+export function is<TSchema extends StructSchema>(x: unknown, schema: TSchema): x is SharedTreeObject<TSchema>;
+
 // @alpha
```

Comment:
Most of this change is just surfacing FieldProxy, NodeProxy, and their dependencies.

...plus 'is(..)' because it's awesome.